extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/MySqlCdcListenBeforeExistIntegrationTest.java

```diff
@@ -54,6 +54,7 @@ public void listenBeforeDatabaseExists() throws Exception {
 
         StreamSource<ChangeRecord> source = sourceBuilder("cdcMysql")
                 .setDatabaseWhitelist(DATABASE)
+                .setCustomProperty("snapshot.mode", "never")
```

Comment:
I have missed this one, why is this change needed?

Reply:
It's because we are inserting the rows just after connector started, we don't really know if the SYNC phase was done - and because of that, the result once will be INSERT, once SYNC. 

If we disable the initial snapshot, we will always have this record as INSERT, otherwise the snapshot process may read it before core binlog reader and report this as SYNC event.

Adding snapshot.mode = NEVER means that the snapshot process won't drop INSERT event in favor of SYNC, so the rest is predictable